plugins/accelerated-peft/src/fms_acceleration_peft/framework_plugin_autogptq.py

```diff
@@ -202,11 +221,19 @@ def augmentation(
     ):
         # guarded imports
         # Third Party
-        from auto_gptq.nn_modules.qlinear.qlinear_tritonv2 import QuantLinear #pylint: disable=import-outside-toplevel,import-error
-        from auto_gptq.utils.peft_utils import GPTQLoraModel, get_gptq_peft_model #pylint: disable=import-outside-toplevel,import-error
+        from auto_gptq.nn_modules.qlinear.qlinear_tritonv2 import (  # pylint: disable=import-outside-toplevel,import-error
```

Comment:
@achew010 why did you change these lines? you mean it wasnt properly linted before?